documentation/modules/post/osx/gather/vnc_password_osx.md

```diff
@@ -0,0 +1,35 @@
+This module show Apple VNC Password from Mac OS X High Sierra.
+
+## Vulnerable Application
+
+  * macOS 10.13.6*
+
+
+## Verification Steps
+
+  Example steps in this format (is also in the PR):
+
+  1. Start `msfconsole`
+  2. Do: `osx/gather/vnc_password_osx`
```

Comment:
This should be:

Do: `use post/osx/gather/vnc_password_osx`

Also, traditionally step 2 is:

`2. Get a session`